modules/flight_interface/flight_interface.py

```diff
@@ -2,13 +2,16 @@
 Creates flight controller and produces local drone odometry coupled with a timestamp.
 """
 
-from . import conversions
-from .. import drone_odometry_local
+from modules import drone_odometry_local
+from modules import decision_command
 
 from ..common.mavlink.modules import drone_odometry
 from ..common.mavlink.modules import flight_controller
 
 
+from . import conversions
```

Comment:
take a look at the convention for imports https://uwarg-docs.atlassian.net/wiki/spaces/CV/pages/2253226033/Python+Style+Convention#Imports